tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -471,8 +450,21 @@ module.exports = {
     getUserSharePostfix: function () {
       return userSharePostfix
     },
-    assertAutocompleteListIsNotVisible: function () {
-      return this.waitForElementNotVisible('@sharingAutoCompleteDropDown')
+    /**
+     * function checks whether autocomplete list is visible
+     *
+     * @returns {Promise<boolean>}
+     */
+    isAutocompleteListVisible: async function () {
+      let isVisible
+      await this.waitForElementNotVisible({
```

Comment:
I think it is better to use this.element instead of `waitForElementNotVisible()`

Reply:
okey-dokey!
:rocket: